cydaf/tomatoClockForPractice
# Report task/api

# why and what
將 task、report api 修正成透過 token 判斷，主要新增：
1. 修改 Task.us -> 增加 user_id 欄位
2. report、task 需判斷是否有帶 token 進入
3. 將 routing 中的 asyncHandler 模組化，檔案為 server/src/common/utils.ts
# How to test
1. set up env file
2. register 輸入 email、name、password
3. verify 輸入 id、verifiedCode
4. login 輸入 email、password -> 複製 data 至各個 token 欄位
5. 測試 task api 以及 report api（將 token 設置在 header 並且將第四步驟之 data 設置為 token 的值）
